lib/main.dart

```diff
@@ -105,8 +93,14 @@ class _MyHomePageState extends State<MyHomePage> {
           // wireframe for each widget.
           mainAxisAlignment: MainAxisAlignment.center,
           children: <Widget>[
-            const Text(
-              'You have pushed the button this many times:',
+            Localizations.override(
+              context: context,
+              locale: const Locale('pl'),
+              child: Builder(
+                builder: (context) {
+                  return Text(AppLocalizations.of(context)!.helloWorld);
```

Comment:
In my opinion it's good idea to create some handy and clean extension to avoid this rather long syntax. 
In toPWR we have sth like this: 

```dart 
 extension BuildContextShortcutExtension on BuildContext {
  AppLocalizations get localize {
    return AppLocalizations.of(this) ?? AppLocalizationsPl();
  }

  void unfocus() {
    FocusScope.of(this).unfocus();
  }
}
```
And the use it like this: 
```dart 
Text(context.localize.helloWorld)
```

Reply:
Yep, but we've used `unfocus` maybe once, but the `localize` is very handy in my opinion

Reply:
I think it's a good suggestion